### ISSUE TITLE
Checkbox laggy inside a Modal or Dialog (only on ios cordova build)

### ISSUE THREAD

**Issue by MoacirSchmidt** [quasarframework/quasar#1711]
Software version

Quasar: 0.15.6
iPhone 7
IOS 11.2.6

Clicking in a checkbox (inside a Modal or Dialog) is lagging when app is built over cordova.
When same app runs over Safari there is no lag!

Please see attached video (at full screen). Running Quasar-Play Dialog showcase (Multiple Choise Selection)

1) Notice the lag (app built with cordova)
2) Changint to safari: There is no lag when change to safari 
3) Back to cordova version 

Video URL: https://youtu.be/PoEzPrXD9gI

**Comment by MoacirSchmidt** [comment]
I noticed that FastClick dependency was removed on quasar 0.14.1 version...Perhaps it is still needed in Dialogs?

**Comment by MoacirSchmidt** [comment]
It seems there is a general smoothy problem when inside a Modal or Dialog! Buttons (and all other "clickable" components) need two clicks before responding. Please take a look at this video:

https://youtu.be/voL54xvUeLY

This happens only on IOS mobile built over cordova

**Comment by rstoenescu** [comment]
Can you reproduce this if you install this better webview? `cordova plugin add cordova-plugin-wkwebview-engine` (in your src-cordova folder)

https://github.com/apache/cordova-plugin-wkwebview-engine

**Comment by MoacirSchmidt** [comment]
I have good and bad news!

1) Good news: The first problem (checkbox lag inside modal or dialog) have gone away when using wkwebview-engine. 

2) Bad news: All components inside Dialog or Modal (including checkboxes, buttons, etc...) still need two taps to respond.

It seems that this second problem is related with IOS using first tap to something else.

angular seems to have similar problem:
https://github.com/angular/material2/commit/4183fbc0004a622180e667c245dcf012a394034f

and the solution:
https://github.com/angular/material2/commit/4183fbc0004a622180e667c245dcf012a394034f

**Comment by MoacirSchmidt** [comment]
cordova-plugin-wkwebview-engine also gives me the following error on XCode log output window:

```
ERROR: {"code":18,"name":"SecurityError","message":"Blocked attempt to use history.replaceState()
to change session history URL from 
file:///var/containers/Bundle/Application/5F1825A3-F60F-4793-A2F7-8D3119804B53/Adalberto%20Valle.app/www/index.html#/ 
to file:///var/containers/Bundle/Application/5F1825A3-F60F-4793-A2F7-8D3119804B53/Adalberto%20Valle.app/www/. Paths and fragments must match for a sandboxed document.","line":1,"column":76991,"sourceURL":"file:///var/containers/Bundle/Application/5F1825A3-F60F-4793-A2F7-8D3119804B53/Adalberto%20Valle.app/www/js/vendor.2989499d828f3a4c413c.js"}
```

This erro only occurs when building inside XCode

**Comment by MoacirSchmidt** [comment]
But the "two tap problem" persists....

**Comment by MoacirSchmidt** [comment]
There is another problem on checkbox inside Dialog or Modal: State does not change when click on checkbox Text. Take a look at this video plz:

https://youtu.be/uHDtAOpvyjM

**Comment by rstoenescu** [comment]
I've just fixed the two tap problem on iOS. Available in 0.15.9.
